src/nginxconfig/generators/conf/drupal.conf.js

```diff
@@ -32,6 +34,15 @@ export default global => {
         deny: 'all',
     };
 
+    config['# Allow image styles to be handled by the CMS.'] = '';
+    config['location ~ ^/sites/[^/]+/files/styles/'] = {
+        try_files: '$uri @rewrite',
+    };
+
+    config['location @rewrite'] = {
+        rewrite: '^/(.*)$ /index.php?q=$1',
+    };
```

Comment:
Why does this use `rewrite` and not just internally rewrite with `try_files`?

Wouldn't this achieve the same as these two blocks, without sending a rewrite to the client?

```
location ~ ^/sites/[^/]+/files/styles/ {
    try_files $uri /index.php?q=$uri&$args;
}
```

Reply:
Yes, you are right.